### PR TITLE
Add contributors section

### LINKED ISSUE #146: The About page does not showcase the people who made this beautiful app a reality
**Is your feature request related to a problem? Please describe.**
[**Live footage of me going to the About page and not seeing all ya'll on there in some way**](https://giphy.com/gifs/bbc-bbc-two-versailles-2-51Y804o23LzorasImh)

**Describe the solution you'd like**
10x the recognition

**Describe alternatives you've considered**
Dusting off the old typewriter...

**Additional context**
If people are comfortable with it, it would be cool to have some way to showcase everyone who has contributed to it on here and whenever someone new contributes they are added as well. 

p.s. LOVE THE APP! 
p.s.s. LOVE THE LINE "We believe in Freedom and Independence"

**Story:**
As a user I would like to know who built this app so that I can stalk them to see what else they may be working on.

**Details:**
- Use GH API to get info about contributors to this project dynamically
- Add contributors Avatar and link to their profile on the About page

### PR DESCRIPTION
**Pull request by Soreg**
This PR adds all the contributors who ever pushed code to the repo.
Data is fetched through the Github API.

Resolves #146

### CODE REVIEW

File: components/aboutUs/Contributors.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+const Contributors = () => {
+  const [contributors, setContributors] = useState<any[]>([]);
+
+  useEffect(() => {
+    fetch("https://api.github.com/repos/chingu-voyages/moonshot-chingu-quiz/contributors")
+      .then(response => response.json())
+      .then(json => setContributors(json));
+  }, []);
+
+  return contributors && contributors.length > 0 ? (
```
Comment (Soreg) [inline components/aboutUs/Contributors.tsx:41]: Note: I opted to hide the entire section if no contributors were found, or if the request failed.. better than showing an empty "contributors" section.

**Review by jaytula — APPROVED** [review]
Looks good to me. This is how it looks:

image

File: components/aboutUs/Contributors.tsx (hunk 1 above)
Comment (cMikolai) [reply components/aboutUs/Contributors.tsx:41]: Great idea, hehe

**Review by cMikolai — APPROVED** [review]
Yay!

**Comment by rmcguire6** [comment]
Looks good to me. Thanks Jay for the screenshot. Very efficient.

**Review by rmcguire6 — APPROVED** [review]

**Pull request merged**